### DISCUSSION TITLE
Error after upgrading the Plotly.Blazor nuget version from 1.5.0 to 2.0.0

### DISCUSSION THREAD

**Discussion by alsuomi** [LayTec-AG/Plotly.Blazor#75] [Q&A] [answered]
Hey

After upgrading the nuget package to 2.0.0, it has stopped working.
The references cannot be found even if the Nuget Package's installation is successful.

Can't even downgrade to 1.5.0!!

- The name 'ModeFlag' does not exist in the current context
- The type or namespace name 'Plotly' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'Layout' could not be found (are you missing a using directive or an assembly reference?
- The type or namespace name 'ITrace' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'Config' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'PlotlyChart' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'Scatter' could not be found (are you missing a using directive or an assembly reference?)

Plugin version 1.5.0 is working fine with a NEW blazor project. However, as I mentioned, I can't downgrade anymore.

Please reply and provide some support what to do after the update.

A quick support will be appreciated. Thanks!

**Comment by sean-mcl** [comment, marked as answer]
Hey,

are you using .NET 5?

You can check it in the `* .csproj`-File

```
<Project Sdk="Microsoft.NET.Sdk.Web">
...
  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>
...
</Project>

```

**Reply by alsuomi** [reply]
<TargetFramework>netcoreapp3.1</TargetFramework>

**Reply by sean-mcl** [reply]
Yes, the 2.0.0 package is only for .NET 5 and higher.
I recommend you to move to .NET 5. For this you might have to update your Visual Studio once.

However, if you want to continue using netcoreapp3.1, you can manually change the version in the `.csproj`  file and rebuild it.

If you are using a git repo and don't care about your current unsaved changes, you can also reset your repository with the following command. Please use this command with extreme caution.

`git clean -dfx`
